packages/babel-jest/src/index.ts

```diff
@@ -48,10 +49,10 @@ const createTransformer: CreateTransformer = userOptions => {
   } as const;
 
   function loadBabelConfig(
-    cwd: Config.Path,
     filename: Config.Path,
     transformOptions: JestTransformOptions,
   ): PartialConfig {
+    const {cwd} = transformOptions.config;
     // `cwd` first to allow incoming options to override it
     const babelConfig = loadPartialConfig({
```

Comment:
There is also the async version of this function, and I _think_ it allows using `await` in config files.

Reply:
Ah, I assumed there wasn't as it's not in the types 😅 

![image](https://user-images.githubusercontent.com/1404810/111037243-94cb3800-8423-11eb-8c71-7ea36209b3d3.png)

Reply:
Uh it's because the types are not maintained by us.

We are slowly converting our codebase to TS, we'll release official type definitions when we finish.

Reply:
I can send a PR to DT 👍

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51741